.pre-commit-config.yaml

```diff
@@ -64,7 +64,7 @@ repos:
             --locked,
             --workspace,
             --features,
-            cubeb,
+            cpal,
```

Comment:
This should probably be reverted and investigated separately

Reply:
The problem is that I was even not able to make a commit, because the pre-commit hook was crashing